package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "eslint-config-fluid",
+    "version": "0.1.0",
+    "description": "fluid-project eslint configuration",
+    "main": "index.js",
+    "engines": {
+        "node": ">=4.0.0"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/fluid-project/eslint-config-fluid.git"
+    },
+    "keywords": [
+        "eslint",
+        "eslintconfig",
+        "fluid",
+        "fluid-project"
+    ],
+    "author": "Fluid Project",
+    "license": "BSD-3-Clause",
+    "bugs": {
+        "url": "https://issues.fluidproject.org/projects/FLUID/issues"
+    },
+    "homepage": "http://fluidproject.org",
+    "peerDependencies": {
+        "fluid-eslint": ">=2.0.0"
```

Comment:
As far as I can tell we can track the issue here: https://github.com/ember-cli/ember-cli-eslint/issues/63  - doesn't look like progress is likely